src/app/about/page.tsx

```diff
@@ -1,92 +1,76 @@
-// SPDX-FileCopyrightText: 2024 PNED G.I.E.
-//
-// SPDX-License-Identifier: Apache-2.0
+/* SPDX-FileCopyrightText: 2024 PNED G.I.E. */
 
+/* SPDX-License-Identifier: Apache-2.0 */
+
+"use client";
```

Comment:
question: @zalborzi why does it have to be a client component? You can access the about.md directly in the server side.

Reply:
I went with my gut! I faced several issues. so i changed it to be server side.